content/en/docs/guides/metadata.md

```diff
@@ -48,25 +54,25 @@ Custom metadata must follow the "Custom-Metadata" format listed in
 [PROTOCOL-HTTP2](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md)
 , with the exception of binary headers not needing to be base64 encoded.
 
-
-
 #### Headers
-Headers are sent before any data.  The header includes things like
-authentication credentials and how to handle the RPC.  
 
-Some of the headers, such as authorization are generated by gRPC for you.  
-Custom header handling is language dependent. For example, in Java and Go they
-are generally provided through Interceptors.
+Headers are sent with the initial request message from the client to the server
+and with the initial response from the server to the client. The header includes
+things like authentication credentials and how to handle the RPC.
+Some of the headers, such as authorization, are generated by gRPC for you.  
+Custom header handling is language dependent, generally through interceptors.
 
 #### Trailers
-Trailers are a special kind of header that is sent after the message data.  They
-are used internally to communicate the outcome of an RPC or that an RPC has been
-cancelled.  At the application level, custom trailers can be used to communicate
+
+Trailers are a special kind of header that is sent after the message data. They
+are used internally to communicate the outcome of an RPC. At the application
+level, custom trailers can be used to communicate
 values known only after the message data has been completely generated such as
-digital signatures and integrity checks or anything else the server wants to
+digital signatures and integrity checks, or anything else the server wants to
```

Comment:
Digital signatures and integrity checks are much better served in the messages themselves. Maybe a better example would be server utilization and query cost; those we actually do with ORCA.